### ISSUE TITLE
QGIS plugin not emptied on new project

### ISSUE THREAD

**Issue by deltamarnix** [Deltares/Ribasim#1682]
- Open QGIS and ensure that the Ribasim plugin is installed and enabled.
- Open the application via the Ribasim button on the QGIS toolbar: _Ribasim panel opens_.
- Press the "New" button in the Model tab: _file navigation window pops up_.
- Fill in a name ("test1").
- Press OK: _A toml and database file are created on the given location_.
- Press "Project > New": _Popup asks to save the project_.
- Press Discard: _Layers panel is emptied, but Ribasim tab is still filled with the layers and the file name_.

**Comment by deltamarnix** [comment]
I expected the Ribasim tab to be emptied, because we are starting a whole new project.

**Comment by Huite** [comment]
I agree that makes more sense. If you define a reset method in the plugin, you can connect it with: `QgsProject.instance().cleared`

I'm not sure if `cleared` signals on opening an existing project. In that case you'd listen to QgsProjeect.instance().read_project`, I reckon.
